target-riscv/cpu.h

```diff
@@ -80,6 +80,7 @@ struct CPURISCVState {
     uint32_t amoinsn;
     target_long amoaddr;
     target_long amotest;
+    target_long rve;
```

Comment:
This patch doesn't implement the "illegal instruction for register specifiers > 15" part of RV32E.  When that part is implemented, we'll need tb_flags to allow the RV32E to affect translation in a controlled way; without that part, either option works.